src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -91,6 +91,11 @@ kibana_vars=(
     apm_oss.transactionIndices
     apm_oss.metricsIndices
     xpack.canvas.enabled
+    xpack.code.ui.enabled
+    xpack.code.indexRepoFrequencyMs
```

Comment:
just curious, do all flags in Code need to be generated? If so, there should be more. Or just the critical `xpack.code.ui.enabled`?

Reply:
all would be ideal

Reply:
Just enumerate all flags which are available to end-users below:

```
xpack.code.ui.enabled
xpack.code.indexRepoFrequencyMs
xpack.code.updateRepoFrequencyMs
xpack.code.lsp.verbose
xpack.code.verbose
xpack.code.security.gitHostWhitelist
xpack.code.security.gitProtocolWhitelist
xpack.code.security.enableGitCertCheck
xpack.code.maxWorkspace
xpack.code.disk.thresholdEnabled
xpack.code.disk.watermarkLow
```

Reply:
the configs of this setting doc have not been updated yet. The ones that I listed here has a couple of new ones which will be included in 7.4+